README.md

```diff
@@ -26,10 +26,11 @@ and the storage logic.
 
 ## Infrastructure
 
-The sample application can be deployed in two different ways:
+The sample application can be deployed in three different ways:
 1. JVM - This mode uses zip packaging style and runs on the JVM inside the Lambda environment.
 2. Custom Runtime - This mode uses GraalVM native image and uses Custom Runtime to run on Lambda. 
    GraalVM native image is a stand-alone execution binary and does not require a JVM to run.
+3. SnapStart - This mode enables SnapStart on the Lambda functions. Current examples only included Spring boot version of the application. Stay tuned for other framework examples with SnapStart. Learn more about SnapStart [here](https://docs.aws.amazon.com/lambda/latest/dg/snapstart.html).
```

Comment:
JVM (without SnapStart)
JVM (with SnapStart)
Custom Runtime

in that order seems like a good progression

Reply:
Done